### ISSUE TITLE
Find a good name for the Knora API server

### ISSUE THREAD

**Issue by benjamingeer** [dasch-swiss/knora-api#521]
"Knora API server" is too long and boring. Let's think of a nice name. Preferably one that:

* Sounds friendly
* Isn't already used for anything on the Internet
* Doesn't mean anything horrible in any language

**Comment by loicjaouen** [comment]
what about 'knapi' ?  

The only website I find is about a subject that is loosely related and probably not quite active ( http://www.knapi.com/ ).

About the meaning, I don't know how to find out, but asking google translate, it detects 'knapi' as Letton translating into 'barely' which is not very positive but we could make sense of it.

https://translate.google.com/#auto/en/knapi

we can also find some islandic roots:
http://conceptnet5.media.mit.edu/web/c/is/knapi

where it means 'valet', 'knight' or 'rider'... which sounds more positive and fitting.

**Comment by benjamingeer** [comment]
"Knapi" would be pronounced "nappy", which is this :)

![lx6049_1](https://cloud.githubusercontent.com/assets/558389/26630385/bb8eaf1c-4606-11e7-88d4-ed71476e3824.jpeg)

**Comment by loicjaouen** [comment]
right, stupid me! you don't want to put your data in a disposable thing made to hold poo
Quite lovely though, even with a quite wild gaze.

**Comment by sofiag** [comment]
How about  	

KAISER 	  Knora ApI SERver

It's an emperor and ok, it's also a bread and a beer, but it's all good stuff, right? ;-)

**Comment by benjamingeer** [comment]
It would be nice if you could do a google search for the name and get our project as one of the first few search results. :) This is why I suggested it should be something that isn't already used for anything on the web.

**Comment by sofiag** [comment]
How about  	
KAISER 	  Knora ApI SERver
It's an emperor and ok, it's also a bread and a beer, but it's all good stuff, right? ;-)

Or KNAP KNora APi 
Knap means "good (looking)" in dutch, and "button" in Danish... In English it stands for "the crest of a hill"...?

**Comment by sofiag** [comment]
I cannot imagine any word/letter combination that is not used for something on the web already...
:-D
ok, I'll try harder...

**Comment by benjamingeer** [comment]
"KNAP" sounds like it's about taking a nap. 😴   😀

**Comment by sofiag** [comment]
haha that's  also not necessarily a bad thing ;-)

**Comment by benjamingeer** [comment]
Some strategies:

* Take two words that are hardly ever used together, and combine them to make one word. That's how I named http://freemarker.org/ .
* Take something that sounds like a person's name (Nora) and misspell it so that the result isn't an existing word. That's how we got Knora. I think we did pretty well, because when you do a Google search for "Knora", the first six search results point to us.
